client/pkiclient/katzenmint.go

```diff
@@ -122,14 +124,31 @@ func (p *PKIClient) GetDoc(ctx context.Context, epoch uint64) (*cpki.Document, [
 		p.log.Warningf("Get() returned pki document for wrong epoch: %v", doc.Epoch)
 		return nil, nil, s11n.ErrInvalidEpoch
 	}
-	p.log.Debugf("Document: %v", doc)
+
+	s := "Topology: [0]{"
```

Comment:
IMHO, we need a debug flag either for building or runtime, so that we can show this kind of debug message.